### PR TITLE
feat(builder): add kubectl and run slugbuilder as a pod

### PR DESCRIPTION
**Pull request by smothiki**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 rootfs/bin/boot
+rootfs/usr/bin/*
```
Comment (mboersma) [inline .gitignore]: Isn't this already covered by the `rootfs/usr/bin/` exclusion below?

Comment (smothiki) [inline .gitignore]: opps My bad will remove it. But even though there is a git ignore endpoint while doing git push its still considering binaries in that direcotry

**Comment by smothiki** [comment]
closing this in favor of  https://github.com/deis/builder/pull/8

**Comment by arschles** [comment]
@smothiki can you cherry pick commits from https://github.com/smothiki/builder/pull/1 into #8

**Pull request closed without merging**